modules/apps/portal-security/portal-security-content-security-policy/src/main/java/com/liferay/portal/security/content/security/policy/internal/servlet/taglib/ContentSecurityPolicyTopHeadDynamicInclude.java

```diff
@@ -57,8 +71,35 @@ public void register(DynamicIncludeRegistry dynamicIncludeRegistry) {
 		dynamicIncludeRegistry.register("/html/common/themes/top_head.jsp#pre");
 	}
 
+	private ContentSecurityPolicyConfiguration
+		_getContentSecurityPolicyConfiguration(
+			HttpServletRequest httpServletRequest) {
+
```

Comment:
This is the same method we have [here](https://github.com/liferay-frontend/liferay-portal/pull/3793/files#diff-fcc978b2491286d2c77bcb694142b53d03d75869d9989f02c2e6d6e505ccfc3aR93), we'll need some sort of utility to host this one